dj_cqrs/management/commands/cqrs_deleted_sync_replica.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright © 2021 Ingram Micro Inc. All rights reserved.
+
+import sys
+
+import ujson
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import DatabaseError
+
+from dj_cqrs.registries import ReplicaRegistry
+
+
+class Command(BaseCommand):
+    help = 'Diff for deleted objects synchronizer from CQRS master stream.'
+
+    @classmethod
+    def deserialize_in(cls, package_line):
+        return ujson.loads(package_line)
+
+    def handle(self, *args, **options):
+        with sys.stdin as f:
+            first_line = f.readline().strip()
+            model = self._get_model(first_line)
+
+            for pks_line in f:
+                try:
+                    model._default_manager.filter(
+                        pk__in=self.deserialize_in(pks_line.strip()),
```

Comment:
Let's start with this one. But I know places where in replica we have different pk or complex pk's